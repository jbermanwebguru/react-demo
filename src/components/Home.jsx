import React from 'react';
import { Link } from 'react-router-dom';

const Home = () => {
  return (
    <div>
      <h1>Player Roster</h1>

      <Link className="button" to="/login">Login</Link>

      <Link className="button" to="/register">Register</Link>
    </div>
  );
};

export default Home;
