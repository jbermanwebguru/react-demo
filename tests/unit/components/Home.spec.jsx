import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import configureStore from 'redux-mock-store';

import Home from '../../../src/components/Home';

describe('<Home />', () => {
  it('renders the component', () => {
    const wrapper = shallow(<Home />);
    const component = wrapper.shallow();

    expect(toJson(component)).toMatchSnapshot();
  });
});
