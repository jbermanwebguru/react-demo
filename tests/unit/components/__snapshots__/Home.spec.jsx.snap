// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> renders the component 1`] = `
<div>
  <h1>
    Player Roster
  </h1>
  <Link
    className="button"
    replace={false}
    to="/login"
  >
    Login
  </Link>
  <Link
    className="button"
    replace={false}
    to="/register"
  >
    Register
  </Link>
</div>
`;
